NoteBerkayMvvm/Scenes/Auth/ForgotPasswordConfirm/ForgotPasswordConfirmViewModel.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ForgotPasswordConfirmViewModel.swift
+//  NoteBerkayMvvm
+//
+//  Created by Mustafa Berkay Kaya on 22.02.2022.
+//
+
+import Foundation
+
+protocol ForgotPasswordConfirmViewDataSource {
+    var email: String { get set }
+}
+
+protocol ForgotPasswordConfirmViewEventSource {}
+
+protocol ForgotPasswordConfirmViewProtocol: ForgotPasswordConfirmViewDataSource, ForgotPasswordConfirmViewEventSource {
+    
+    func pushLoginScreen()
+}
+
+final class ForgotPasswordConfirmViewModel: BaseViewModel<ForgotPasswordConfirmRouter>, ForgotPasswordConfirmViewProtocol {
+    var email: String = ""
+    
+    init(email: String, router: ForgotPasswordConfirmRouter) {
+        super.init(router: ForgotPasswordConfirmRouter())
```

Comment:
router'ı zaten route'dan verdiğin için direk initilizeda verdiğin router'ı kullanabilirsin. Yani;
  super.init(router: router)
şeklinde